### ISSUE TITLE
Leatherback Snapjaw: no drop limit

### ISSUE THREAD

**Issue by Gasan918** [slowtorta/turtlewow-bug-tracker#5252]
**Please, provide a clear description what the bug is:**

Leatherback Snapjaw egg keeps dropping even if player have got this item/companion already.

![image](https://github.com/slowtorta/turtlewow-bug-tracker/assets/96973359/6026307f-604d-486c-a92d-5bf8033ac135)

**Steps to reproduce the behavior with as much detail as possible:**

1. Get one egg from turtle. 
2. Go on killing turtles
3. Get more and more eggs

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

Item: https://database.turtle-wow.org/?item=18966

**Expected behavior. Describe how it should work:**

One egg is enough.

**Comment by twowTurtReynolds** [comment]
Indeed. Not sure its possible to have these not drop when already learned. Best course might be to flag unique so only 1 of these extra ones can fill your bags at any given time.

**Comment by Dragunovi** [comment]
Changed all turtle companion egg drops to be unique. Should be applied with the next scheduled restart.
